lib/klife/connection/broker.ex

```diff
@@ -75,6 +75,10 @@ defmodule Klife.Connection.Broker do
     {:noreply, state}
   end
 
+  def handle_info({ref, _resp}, %__MODULE__{} = state) when is_reference(ref) do
+    {:noreply, state}
+  end
```

Comment:
Since we are using `start_child` this is not necessary. Because the callback will not be linked to the broker connection process in any way.